src/Components/Admin/UserManagement.jsx

```diff
@@ -31,7 +35,7 @@ const Wrapper = styled.div`
 `;
 
 // Moock data
-const users = new Array(30).fill(0).map((elem, index) => ({
+const users2 = new Array(30).fill(0).map((elem, index) => ({
```

Comment:
Fixed array size of 30 - not sure if this is okay or not.

Reply:
I also question the variable name. Why was it changed from users to users2? And was the usage updated throughout? (ie. users2 isn't used anywhere else in this file)